### PR TITLE
modified send-error to take more than one error

### PR DESCRIPTION
**Pull request by ShurakaiSoft**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="http://app.review.ninja/agco/harvesterjs/pull/92" target="_blank">ReviewNinja

File: lib/send-error.js [hunk 1]
```diff
@@ -25,18 +35,26 @@ var sendError = function (req, res, error) {
 
     try {
 
-        /** todo handle possibility of array of errors in case of a bulk insert
-         * right now the promise chain of createResources fails on the first error
-         * the use of https://github.com/tildeio/rsvp.js/#all-settled-and-hash-settled can improve this
-         * and return back errors for all rejected items
-         * */
+        // premote error if it's not already an Array
```
Comment (ssebro) [inline lib/send-error.js]: promote :P

File: lib/send-error.js [hunk 2]
```diff
@@ -4,7 +4,17 @@ var _ = require('lodash');
 
 var JSONAPI_Error = require('./jsonapi-error');
 
+/**
+ * Send a JSONAPI compatible list of errors to the client.
+ *
```
Comment (ssebro) [inline lib/send-error.js]: Nice docs!

File: lib/send-error.js [hunk 3]
```diff
@@ -25,18 +35,26 @@ var sendError = function (req, res, error) {
...
-         * the use of https://github.com/tildeio/rsvp.js/#all-settled-and-hash-settled can improve this
-         * and return back errors for all rejected items
-         * */
+        // premote error if it's not already an Array
+        if (!(error instanceof Array)) {
+            error = [ error ];
+        }
+
+        // handle each error
+        _.forEach(error, function A(error, index, collection) {
```
Comment (ssebro) [inline lib/send-error.js]: Did you name the function A?

**Comment by ssebro** [comment]
Awesome!

**Pull request merged**